nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment-api/src/main/java/org/nuxeo/ecm/platform/comment/api/CommentConstants.java

```diff
@@ -28,7 +28,7 @@
 
     public static final String EVENT_COMMENT_CATEGORY = "commentCategory";
 
-    public static final String PARENT_COMMENT = "parentComment";
+    public static final String COMMENTED_DOCUMENT = "parentComment";
```

Comment:
missing @since 11.1 unless you are 100% sure this constant is not used by customer code

Reply:
generally we don't rename a public/protected constants, what we do it create a new one, deprecate the old one and replace the callers of the this old one by the new one

Reply:
I'll go for the new constant then

Reply:
This has been droped in the last push -f
We now have a `TOP_LEVEL_COMMENT` which is used to keep track of the original document that is commented and needs to be linked in notification emails from the hypothetic parent comment